Formula/datomic.rb

```diff
@@ -21,13 +21,14 @@ def install
 
     # install free-transactor properties
     data = var/"lib/datomic"
-    (etc/"datomic").mkpath
-    (etc/"datomic").install libexec/"config/samples/free-transactor-template.properties" => "free-transactor.properties"
 
-    inreplace "#{etc}/datomic/free-transactor.properties" do |s|
+    inreplace "#{libexec}/config/samples/free-transactor-template.properties" do |s|
       s.gsub! "# data-dir=data", "data-dir=#{data}/"
       s.gsub! "# log-dir=log", "log-dir=#{data}/log"
     end
+
+    (etc/"datomic").mkpath
```

Comment:
I don't think this `mkpath` was ever actually needed

Reply:
ah all right then, I remove it.